github_wrapper/api_wrapper/__init__.py

```diff
@@ -0,0 +1 @@
+from .gwrapper import GWrapper

```

Comment:
Try removing this line and run? IS it necessary to place this import here?

Reply:
Otherwise it throws NameError exception when initializing GWrapper instance.